recipes/magics/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "2.34.1" %}
+
+package:
+  name: magics
+  version: {{ version }}
+
+source:
+  fn: Magics-{{ version }}-Source.tar.gz
+  url: https://software.ecmwf.int/wiki/download/attachments/3473464/Magics-{{ version }}-Source.tar.gz
+  sha256: 8df27f8f262ebc32a61f8696df15a7b4a6e4203b2a8e53fe7aa13caa1c4e3fa4
+
+build:
+  number: 0
+  skip: True  # [win or not py27]
+  detect_binary_files_with_prefix: true
+
+requirements:
+  build:
+    - cmake
+    - swig
+    - boost 1.64.*
+    - proj.4 4.9.3
+    - python-eccodes
```

Comment:
Argh. Just always bites us b/c both `defaults` and `conda-forge` used to name it `proj.4`...